hw2/03-count.js

```diff
@@ -1,3 +1,16 @@
-// Add your code here
+const textInput = document.getElementById('input');
+const text = document.getElementById('text');
+const origText = text.innerHTML;
 
-input.addEventListener('keydown', handleKeyDown);
+textInput.addEventListener('keydown', handleKeyDown);
+
+function handleKeyDown(event) {
```

Comment:
Also nitpick. I feel the thing should search the whole thing while entering each key, and not wait till enter key. But also not given in the spec so... Just a sugestion. Personally like the continual search more. But it does not matter

Reply:
Still a good impl here. Only exec if we hit an enter key, works well. Good job!